entities/accounts/edit/general-information/general-information.module.scss

```diff
@@ -2,4 +2,8 @@
     display: flex;
     flex-direction: row;
     gap: 40px;
+
+    box-sizing: border-box;
```

Comment:
```css box-sozing: borer-box``` already implemented in `_boilerplate.css`.

Reply:
deleted